src/javascripts/helpers/validationForm.js

```diff
@@ -64,8 +64,15 @@ export const validateTransaction = (data) => {
     },
   ]
 
-  for(const key in data){
-    const value = data[key];
+  const transactionCheck = {
```

Comment:
Should we do this? I thought data already have its, no need to map fields here

Reply:
I fixed @phuocduong-agilityio . Please help me check. Thanks!